packages/webapp/__tests__/AccountSecurityPage.tsx

```diff
@@ -158,8 +158,17 @@ it('should allow changing of email', async () => {
   };
   mockWhoAmIFlow(email);
   mockSettingsValidation(params);
-  const submitChanges = await screen.findByText('Send code');
-  fireEvent.click(submitChanges);
+
+  const firstSubmitButton = await screen.findByText('Send code');
+  const mockFirstSubmit = new Event('submit', {
+    bubbles: true,
+    cancelable: true,
+  });
+  Object.defineProperty(mockFirstSubmit, 'submitter', {
+    value: firstSubmitButton,
+  });
```

Comment:
React Testing Library does not replicate a "real" form submission event, meaning some properties get omitted, such as submitter. One way to solve that is to add the add the event manually.

Reply:
Oh, wow. I did not know about this one. Great info.